core/tests/test_suite.py

```diff
@@ -0,0 +1,114 @@
+# coding: utf-8
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Oppia-ml test suite.
+
+In general, this script should not be run directly. Instead, invoke
+it from the command line by running
+
+    bash scripts/run_backend_tests.sh
+
+from the oppia/ root folder.
+"""
+
+# Pylint has issues with import order of argparse.
+#pylint: disable=wrong-import-order
+import argparse
+import os
+import sys
+import unittest
+#pylint: enable=wrong-import-order
+
+import vmconf
+
+CURR_DIR = os.path.abspath(os.getcwd())
+THIRD_PARTY_DIR = os.path.join(CURR_DIR, 'third_party')
+MANIFEST_FILE_PATH = os.path.join(os.getcwd(), 'manifest.txt')
+
+with open(MANIFEST_FILE_PATH, 'r') as f:
+    DIRS_TO_ADD_TO_SYS_PATH = [
```

Comment:
Yeah this code is repeating again. There is a function `configure()` for this in `vm_config.py`. Should I use it in this file?